articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+
+   - **Shared Key** - A mixture of letters and numbers, used to establish encryption for the connection. You must use the same shared key in both the virtual network and local network gateways. If your gateway device doesn't provide one, you can make one up here and provide it to your device.
+
+   - **Subscription** - This is your UKCloud for Microsoft Azure subscription.
+
+   - **Resource Group** - When adding a connection for an existing peer, the resource group can't be changed.
+
+   - **Location** - Select the location of the Azure resource.
+
+     ![Add new connection](images/azs-public-browser-add-connection-blade.png)
+
+6. Click **OK**.
+
+## Update the local Gateway IPs in Azure Stack and Public Azure
+
+In order to establish the connection you will need to identify the public IPs that you created in Azure Stack and Public Azure, and then update both local gateways.
+
+### Update the local gateway in Public Azure
+
+1. In the Azure Stack portal, navigate to the *connections* blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
+
+    ![All services connections networking section](images/azs-browser-all-services-connections-networking.png)
+
+2. Select the VPN connection you created in the earlier stages.
+
+    ![VPN connections](images/azs-browser-vpn-connection-view.png)
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
+
+    ![VPN connection public IP](images/azs-browser-site-to-site-vpn-connection-public-ip.png)
+
+4. Log in to the [Public Azure portal](https://portal.azure.com).
+
+5. In the Public Azure portal, navigate to the *Local Network Gateway* blade by clicking **All services** in the favourites panel, then selecting **Local network gateways** under the *networking* section.
+
+    ![Azure Public virtual network button](images/azs-public-browser-services-virtual-network-gateway-button.png)
+
+6. Select the local network gateway that you created in the earlier steps.
+
+7. In the local gateway blade, under *Settings*, select **Configuration**.
+
+    ![Azure Public local network gateway](images/azs-public-browser-local-gateway-configuration-button.png)
+
+8. Change the **IP address** field to the IP taken from the Azure Stack connection.
+
+    ![Azure Public local gateway public IP address](images/azs-public-browser-local-gateway-ip.png)
+
+9. Click **Save**.
+
+### Update the local Gateway IPs in Azure Stack
+
+1. In the Public Azure portal, navigate to the Connections blade by clicking **All services** in the favourites panel, then selecting **Connections** under the *networking* section.
+
+    ![Connections button under all services in public Azure](images/azs-public-browser-services-connections-button.png)
+
+2. Select the VPN connection you created in the earlier stages.
+
+3. Make note of the public IP this can be found in the *Virtual network gateway* section.
```

Comment:
Make a note of the public IP. You can find this in the *Virtual network gateway* section.